src/worker.js

```diff
@@ -8,13 +8,53 @@
 // This is the entry point file that is loaded first by each Web Worker
 // that executes pthreads on the Emscripten application.
 
+'use strict';
```

Comment:
I thought code that ran in workers (like modules) was implicitly running in strict mode.  I guess I got my wires cross somewhere?

Reply:
No, it's not. By default Workers are just regular scripts, although there is a recent feature where you can do `new Worker(..., { type: 'module' })` and then it will run in a strict module context.